jwt/jwt.go

```diff
@@ -0,0 +1,37 @@
+package jwt
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/golang-jwt/jwt"
+)
+
+var secretKey = []byte("farawin")
+
+func CreateToken(id string) (string, error) {
+	token := jwt.NewWithClaims(jwt.SigningMethodHS256, jwt.MapClaims{
+		"id":  id,
+		"exp": time.Now().Add(time.Hour * 24).Unix(),
+	})
+	tokenString, err := token.SignedString(secretKey)
+	if err != nil {
+		return "", err
```

Comment:
If you are using a function from another package, it's very **_RECOMMENDED_** to add context to your errors.
You can achieve that by using fmt.Errorf function like `return "",fmt.Errorf("error getting signed string from token: %w",err)`.
It also helps you to wrap your errors. you can search about wrapping errors later.